crypto/multisig/compact_bit_array.go

```diff
@@ -71,6 +71,21 @@ func (bA *CompactBitArray) SetIndex(i int, v bool) bool {
 	return true
 }
 
+// NumOfTrueBitsBefore returns the location of the given index, among the
+// values in the bit array that are set to true.
+// e.g. if bA = _XX_X_X, NumOfTrueBitsBefore(4) = 2, since
+// the value at index 4 of the bit array is the third
+// value that is true. (And it is 0-indexed)
+func (bA *CompactBitArray) NumOfTrueBitsBefore(index int) int {
```

Comment:
minor point, I'd just call this `NumTrueBitsBefore(index int)`.  I think "of" is implied.